R/tostring.R

```diff
@@ -152,8 +152,10 @@ setMethod("toString", "MatrixPrintForm", function(x,
   # Check for which row has indent
   ind_from_strings <- nchar(old_indent)[-seq_len(nlh)] > 0
   if (!all(ind_from_strings == ind_from_mf)) {
-    stop("Row-info and string indentations are different.", # nocov
-         " Please contact the maintainer, this should not happen.") # nocov
+    stop(
+      "Row-info and string indentations are different.", # nocov
+      " Please contact the maintainer, this should not happen."
+    ) # nocov
```

Comment:
@gmbecker should I revert all style-related changes? some were automatic imo. The ones about export I did because I think it is more readable if internal/function-related-info stays close to the function definition while the rest is user-facing documentation and should be together, above. It is a change in style, but not in the final docs. I agree anyway that having a uniform style is best. If you want I can revert also these ones.